### ISSUE TITLE
"Remove graph" link should be close to query area

### ISSUE THREAD

**Issue by hagen1778** [prometheus/prometheus#5496]
## Proposal
Move "Remove graph" link to the top of query section on Prometheus `/graph` page.

**Use case. Why is this important?**
Sometimes badly composed query could return a lot of result rows. To remove that result user needs to scroll to the bottom of the page and press on "Remove graph" link. It could be a problem is query returned couple of thousands of result rows.

*“Nice to have” is not a good use case. :)*
It would be nice to place that link near to query area, so user could clear results without scrolling the page.

**Comment by cboggs** [comment]
I agree with your point, and would love to add that the current positioning makes it entirely too easy to accidentally remove the _wrong graph_, as it sometimes looks like it should be tied to the graph *below,* not above, the button.

**Comment by roidelapluie** [comment]
@juliusv what is your opinion here?

**Comment by juliusv** [comment]
If someone can make a design mockup in the new React UI that looks not too cluttered (with query stats displayed and all), then I'm all for it.

**Comment by Harkishen-Singh** [comment]
I think this involves changes in the react UI as well, hence it should be open for now. Once the react work is done, we can consider this closing.

**Comment by beorn7** [comment]
Yes, you are right. This was auto-closed via #5990.

**Comment by beorn7** [comment]
Hello from the bug scrub,

Weirdly, the "remove graph" link is still in a very counter-intuitive position. Should be easy to fix, right? Maybe it will change anyway with the upcoming UI revamp? @juliusv or @Nexucis might know the current state.

**Comment by juliusv** [comment]
Yes, with the new UI, it'll be an option in a submenu that is part of the expression input field. Although it's not publicly available yet, I'd already consider it "fixed in the new code", so I'll close this already.

![image](https://github.com/prometheus/prometheus/assets/538008/1609c0b0-70bc-4f21-8413-da89a655d7a6)
